libraries/chain/transaction_context.cpp

```diff
@@ -402,6 +407,26 @@ namespace eosio { namespace chain {
       }
    }
 
+   std::string transaction_context::get_tx_cpu_usage_exceeded_reason_msg(fc::microseconds& limit) const {
+      switch( tx_cpu_usage_reason ) {
+         case tx_cpu_usage_exceeded_reason::account_cpu_limit:
```

Comment:
`limit` is not set for this case.

Reply:
That is the intention. We could report the limit, but it is redundant because that is what is hit.

```
 estimated CPU time (148164 us) is not less than the maximum billable CPU time for the transaction (146537 us) reached account cpu limit
```
It would just report `146537` as the account limit.

I could add a comment to make that clear or I could go ahead and add the redundant limit.

Reply:
Hmm, maybe they are all redundant:

```
transaction was executing for too long 150083us reached on chain max_transaction_cpu_usage 150000us
transaction was executing for too long 10099us reached speculative executed adjusted trx max time 10000us
transaction was executing for too long 25109us reached on chain max_transaction_cpu_usage 24999us
transaction was executing for too long 5104us reached node configured max-transaction-time 5000us
transaction was executing for too long 25068us reached node configured max-transaction-time 25000us

billed CPU time (150001 us) is greater than the maximum billable CPU time for the transaction (150000 us) reached on chain max_transaction_cpu_usage 150000us
billed CPU time (5001 us) is greater than the maximum billable CPU time for the transaction (5000 us) reached trx specified max_cpu_usage_ms 5000us
billed CPU time (131031 us) is greater than the maximum billable CPU time for the transaction (131030 us) reached account cpu limit
estimated CPU time (27900 us) is not less than the maximum billable CPU time for the transaction (27900 us) with a subjective cpu of (131818 us) reached account cpu limit
estimated CPU time (27900 us) is not less than the maximum billable CPU time for the transaction (27900 us) with a subjective cpu of (131818 us) reached account cpu limit
estimated CPU time (148164 us) is not less than the maximum billable CPU time for the transaction (146537 us) reached account cpu limit
estimated CPU time (148164 us) is not less than the maximum billable CPU time for the transaction (146537 us) reached account cpu limit
billed CPU time (138 us) is greater than the maximum billable CPU time for the transaction (0 us) reached account cpu limit
```

Reply:
```
billed CPU time (150001 us) is greater than the maximum billable CPU time for the transaction (150000 us) reached on chain max_transaction_cpu_usage 150000us
billed CPU time (5001 us) is greater than the maximum billable CPU time for the transaction (5000 us) reached trx specified max_cpu_usage_ms 5000us
billed CPU time (131031 us) is greater than the maximum billable CPU time for the transaction (131030 us) reached account cpu limit 131030us
estimated CPU time (27900 us) is not less than the maximum billable CPU time for the transaction (27900 us) with a subjective cpu of (131818 us) reached account cpu limit 27900us
estimated CPU time (27900 us) is not less than the maximum billable CPU time for the transaction (27900 us) with a subjective cpu of (131818 us) reached account cpu limit 27900us
estimated CPU time (148164 us) is not less than the maximum billable CPU time for the transaction (146537 us) reached account cpu limit 146537us
estimated CPU time (148164 us) is not less than the maximum billable CPU time for the transaction (146537 us) reached account cpu limit 146537us
billed CPU time (139 us) is greater than the maximum billable CPU time for the transaction (0 us) reached account cpu limit 0us
```